tests/test_basic_splitter.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+from draive import split_text
+
+
+@pytest.fixture
+def sample_text() -> str:
+    with open("tests/data/sample_text.txt") as file:
+        return file.read()
+
+
+@pytest.fixture
+def nested_splitting_expected_result() -> list[str]:
+    with open("tests/data/nested_splitting.txt") as file:
+        return file.read().split("\n")
```

Comment:
why you are splitting the text here? it is treated as multiple examples at once?

Reply:
the file contains every expected part from a new line :)